xknx/dpt/dpt_1byte_control.py

```diff
@@ -0,0 +1,259 @@
+
+        control, step_code = cls._decode(raw)
+
+        if invert:
+            control = 0 if control > 0 else 1
+
+        return {"control": control, "step_code": step_code}
+
+
+class DPTControlStepwise(DPTControl):
+    """Abstraction for KNX DPT 3.xxx in stepwise mode with conversion to an incement value."""
+
+    dpt_main_number = 3
+    dpt_sub_number = None
+    value_type = "stepwise"
+    unit = "%"
+
+    @staticmethod
+    def _from_increment(value):
+        """Calculate control bit and stepcode as defined in the KNX standard section 3.3.1 from an increment value."""
+        # control bit in KNX standard
+        #   0: - = decrease/move up
+        #   1: + = increase/move down
+        control = 0 if value <= 0 else 1
+
+        stepcode = (
+            0  # special case = break indication (e.g. stop dimming/moving blinds)
+        )
+        if abs(value) >= 100:
+            stepcode = 1
+        elif abs(value) >= 50:
+            stepcode = 2
+        elif abs(value) >= 25:
+            stepcode = 3
+        elif abs(value) >= 12:
+            stepcode = 4
+        elif abs(value) >= 6:
+            stepcode = 5
+        elif abs(value) >= 3:
+            stepcode = 6
+        elif abs(value) >= 1:
+            stepcode = 7
+
+        return {"control": control, "step_code": stepcode}
+
+    @staticmethod
+    def _to_increment(value):
+        """Calculate the increment value from the stepcode and control bit as defined in the KNX standard section 3.3.1."""
+        # calculated using floor(100/2^((value&0x07)-1))
+        inc = [0, 100, 50, 25, 12, 6, 3, 1][value["step_code"] & 0x07]
+        return inc if value["control"] == 1 else -inc
+
+    @classmethod
+    def to_knx(cls, value, invert=False):
+        """Serialize to KNX/IP raw data."""
+        if not isinstance(value, int):
+            raise ConversionError("Cant serialize %s" % cls.__name__, value=value)
+
+        return super().to_knx(cls._from_increment(value), invert)
+
+    @classmethod
+    def from_knx(cls, raw, invert=False):
+        """Parse/deserialize from KNX/IP raw data."""
+        return cls._to_increment(super().from_knx(raw, invert))
+
+
+class DPTControlStepwiseDimming(DPTControlStepwise):
+    """Abstraction for KNX DPT 3.007 / DPT_Control_Dimming in stepwise mode."""
+
+    dpt_main_number = 3
+    dpt_sub_number = 7
+    value_type = "stepwise_dimming"
+
+
+class DPTControlStepwiseBlinds(DPTControlStepwise):
+    """Abstraction for KNX DPT 3.008 / DPT_Control_Blinds in stepwise mode."""
+
+    dpt_main_number = 3
+    dpt_sub_number = 8
+    value_type = "stepwise_blinds"
+
+
+class TitleEnum(Enum):
+    """Enum with a descriptive string representation.
+
+    Ensures values are rendered nicely, e.g. in home assistant.
+    """
+
+    def __str__(self):
+        """Return string representation."""
+        # pylint: disable=no-member
+        return self.name.title()
```

Comment:
Why not use the representation string as Enum.value?
`    DECREASE = "Decrease"`

Reply:
This would require reimplementing the `to_knx` and `from_knx` classes in both `DPTControlStartStopDimming` and `DPTControlStartStopBlinds`, and would not allow a generic implementation in `DPTControlStartStop`. See example in the previous comment above.